pkg/util/files_test.go

```diff
@@ -1,82 +0,0 @@
-// +build unit
-
-/**
- * @license
- * Copyright 2020 Dynatrace LLC
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package util
-
-import (
-	"strings"
-	"testing"
-
-	"gotest.tools/assert"
-)
-
-var appendNameFunc = func(name string) string {
-	return name + "_postfix"
-}
-
-var prependNameFunc = func(name string) string {
-	return "prefix_" + name
-}
-
-var nameReplacingPostfixFunc = func(line string) string {
-	return ReplaceName(line, appendNameFunc)
-}
-
-var nameReplacingPrefixFunc = func(line string) string {
-	return ReplaceName(line, prependNameFunc)
-}
-
-func TestReplaceNameNotMatching(t *testing.T) {
-
-	assert.Equal(t, "management-zone", nameReplacingPostfixFunc("management-zone"))
-	assert.Equal(t, "config:", nameReplacingPostfixFunc("config:"))
-}
-
-func TestReplaceNameMatching(t *testing.T) {
-
-	assert.Equal(t, "- name: test_postfix", nameReplacingPostfixFunc("- name: test"))
```

Comment:
How do you know that the new `RewriteConfigNames` does what its supposed to do without tests similar to these? Or did I miss them in the review? :thinking: